app/controllers/articles_controller.rb

```diff
@@ -28,7 +28,7 @@ def feed
                   handle_tag_feed
                 elsif request.path == latest_feed_path
                   @articles
-                    .where("score > ?", Articles::Feeds::LargeForemExperimental::MINIMUM_SCORE_LATEST_FEED)
+                    .where("score > ?", Articles::Feeds::Latest::MINIMUM_SCORE)
```

Comment:
This constant is now only available in this service which makes more sense given that we specifically check for being on the `last_feed_path` above.